app/views/admin/users/_user.html.erb

```diff
@@ -23,6 +23,9 @@
       <ul class="dropdown-menu dropdown-menu-right" role="menu">
         <li><%= link_to t(:discussions, default: 'Discussions'), admin_user_path(user), remote: true %></li>
         <li><%= link_to t(:edit_user, default: 'Edit User'), admin_user_path(user, mode: 'edit'), remote: true %></li>
+        <li><%= link_to t(:delete_user, default: 'Delete User'), admin_user_path(user), remote: true,
+                        method: :delete,
+                        data: { confirm: "Are you sure you want to delete #{user.name}?" } if current_user.is_admin? %></li>
```

Comment:
@scott & @hin101: As a quick fix, an option is to set those dependent active record associations as dependent: :nullify, which would set the foreign keys to nil when a user is destroyed as opposed to assigning them to a dummy/system user. However, that would not give the admin the control to delete those orphaned records completely. I'm working on a commit to give the admin this option but just wanted to bring up the dependent: :nullify option just in case that's preferable.